manifests/install.pp

```diff
@@ -123,7 +123,7 @@
             fail('manage_package_repo is not supported on Archlinux')
           }
           package { $::grafana::package_name:
-            ensure  => $::grafana::version, # pacman provider doesn't have feature versionable
+            ensure  => $::grafana::package_ensure, # pacman provider doesn't have feature versionable
```

Comment:
Maybe use something like...
```puppet
$ensure = pick($::grafana::package_ensure, $::grafana::version)
package { $::grafana::package_name:
  ensure = $ensure,
...
```
then default your new `package_ensure` parameter to `undef` for OSes other than ArchLinux??